cves/CVE-2005-2970.yml

```diff
@@ -97,22 +110,20 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
+  answer: No evidence found
```

Comment:
Instructions say this can be left blank if no evidence. Feel free to remove

Reply:
Done

Reply:
Content is required to pass Travis so the line was added back into the file